gulpfile.js

```diff
@@ -36,15 +36,12 @@ let liveReloadChangedFiles = [];
 
 function parseCommandLine() {
   commander
-    // Test configuration.
-    // See https://github.com/karma-runner/karma-mocha#configuration
-    .option('--grep [pattern]', 'Run only tests matching a given pattern')
+    .option(
+      '--grep [pattern]',
+      'Run only tests where filename matches a pattern'
+    )
     .parse(process.argv);
 
```

Comment:
I removed the `console.log` here and instead made Karma (in `src/karma.config.js`) print out the matching test files when it starts.